docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -682,6 +682,49 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Local
 Double click the "Add workstations to domain" policy and ensure that the
 "Authenticated Users" group is not present.
 
+## Multiple Domains
+
+Each `windows_desktop_service` is designed to support connecting to hosts in a
+single Active Directory domain. If you have multiple independent domains, you
+can deploy multiple Teleport agents to service them.
+
+If you have multiple domains with a trust relationship between them, you can
+configure Teleport to perform PKI operations against a different domain.
+
+In order for this to work, the hosts that you want to connect to and the AD
+users that you want to connect as must reside in the same domain.
+
+For example, suppose you have a root domain at `example.com` and a child domain
+for developers at `dev.example.com`. If your PKI is configured at the root, but
+you want to allow users in the child domain to connect to hosts in the child
+domain, you would do the following:
+
+First, import Teleport's CA certificate as a trusted root certificate in the root
```

Comment:
Since we're starting this list of instructions after a sentence ending in a colon, I think it would make visual sense to use an ordered list here.
```suggestion
1. Import Teleport's CA certificate as a trusted root certificate in the root
```